src/plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -3,21 +3,25 @@ import moment from 'moment';
 import { IndexPatternMissingIndices } from 'ui/errors';
 import 'ui/directives/validate_index_name';
 import 'ui/directives/auto_select_if_only_one';
-import PluginsKibanaSettingsSectionsIndicesRefreshKibanaIndexProvider from 'plugins/kibana/settings/sections/indices/_refresh_kibana_index';
+import RefreshKibanaIndex from 'plugins/kibana/management/sections/indices/_refresh_kibana_index';
 import uiRoutes from 'ui/routes';
 import uiModules from 'ui/modules';
-import createTemplate from 'plugins/kibana/settings/sections/indices/_create.html';
-
+import createTemplate from 'plugins/kibana/management/sections/indices/_create.html';
 
 uiRoutes
-.when('/settings/indices/', {
-  template: createTemplate
+.when('/management/data/index/', {
+  template: createTemplate,
+  resolve: {
+    indexPatternIds: function (courier) {
+      return courier.indexPatterns.getIds();
+    }
```

Comment:
I don't think this is necessary since you have https://github.com/elastic/kibana/pull/7284/files#diff-36c02af3376ac2e16e88d8b861e0de15R24